### PR TITLE
Cluster analysis

### PR DESCRIPTION
**Pull request by mcreixell**
I've added notes/concerns at the end of the text of each figure (commented out).

### CODE REVIEW

File: manuscripts/method/03.results.md [hunk 1]
```diff
@@ -9,76 +9,110 @@ MS-based global phosphoproteomic data provides unparalleled coverage when interr
...
+
+<!-- Notes figureM2:
+1. C is a more natural position than in D for the heatmap (first show data via PCA and heatmap A-C, then talk about prediction and substreates D-E). This means that the heatmap should be vertical and occupy slots C and E which I don't know how to code. I'll just edit on adobe once we're done reviewing.
```
Comment (aarmey) [inline manuscripts/method/03.results.md:27]: Yep.

File: manuscripts/method/03.results.md [hunk 2]
```diff
@@ -9,76 +9,110 @@ MS-based global phosphoproteomic data provides unparalleled coverage when interr
...
+
+2. I'm showing kinase predictions of CPTAC clusters in C and G so that's why I'm introducing this data but I'm not sure if it's smooth enough (3rd sentence 1st paragraph). 
+
+3. Statistical test in AKT substrates plot?
+
+4. There's a significant difference in ERK2 prediction between CPTAC clusters and ERK2+, which makes sense since all of these clusters have hundreds of peptides, if not thousands, and the ERK2+ only ~20. Maybe this should somehow be mentioned in the text?
+
+5. Shuffled True/False might be unclear?  -->
```
Comment (aarmey) [inline manuscripts/method/03.results.md:35]: Not sure what you mean?

File: manuscripts/method/03.results.md [hunk 3]
```diff
@@ -9,76 +9,110 @@ MS-based global phosphoproteomic data provides unparalleled coverage when interr
...
+
+<!-- Notes figureM2:
+1. C is a more natural position than in D for the heatmap (first show data via PCA and heatmap A-C, then talk about prediction and substreates D-E). This means that the heatmap should be vertical and occupy slots C and E which I don't know how to code. I'll just edit on adobe once we're done reviewing.
+
+2. I'm showing kinase predictions of CPTAC clusters in C and G so that's why I'm introducing this data but I'm not sure if it's smooth enough (3rd sentence 1st paragraph). 
+
+3. Statistical test in AKT substrates plot?
+
+4. There's a significant difference in ERK2 prediction between CPTAC clusters and ERK2+, which makes sense since all of these clusters have hundreds of peptides, if not thousands, and the ERK2+ only ~20. Maybe this should somehow be mentioned in the text?
```
Comment (aarmey) [inline manuscripts/method/03.results.md:33]: Yep. Your explanation here is reasonable.

File: manuscripts/method/03.results.md [hunk 4]
```diff
@@ -9,76 +9,110 @@ MS-based global phosphoproteomic data provides unparalleled coverage when interr
...
+
+<!-- Notes figureM2:
+1. C is a more natural position than in D for the heatmap (first show data via PCA and heatmap A-C, then talk about prediction and substreates D-E). This means that the heatmap should be vertical and occupy slots C and E which I don't know how to code. I'll just edit on adobe once we're done reviewing.
+
+2. I'm showing kinase predictions of CPTAC clusters in C and G so that's why I'm introducing this data but I'm not sure if it's smooth enough (3rd sentence 1st paragraph). 
+
+3. Statistical test in AKT substrates plot?
```
Comment (aarmey) [inline manuscripts/method/03.results.md:31]: To help with this, can this be the actual count, not normalized?

Comment (aarmey) [reply manuscripts/method/03.results.md:31]: After that you can figure out significance using the cdf of a [hypergeometric distribution](https://en.wikipedia.org/wiki/Hypergeometric_distribution).

File: manuscripts/method/03.results.md [hunk 5]
```diff
@@ -9,76 +9,110 @@ MS-based global phosphoproteomic data provides unparalleled coverage when interr
...
+
+<!-- Notes figureM2:
+1. C is a more natural position than in D for the heatmap (first show data via PCA and heatmap A-C, then talk about prediction and substreates D-E). This means that the heatmap should be vertical and occupy slots C and E which I don't know how to code. I'll just edit on adobe once we're done reviewing.
+
+2. I'm showing kinase predictions of CPTAC clusters in C and G so that's why I'm introducing this data but I'm not sure if it's smooth enough (3rd sentence 1st paragraph).
```
Comment (aarmey) [inline manuscripts/method/03.results.md:29]: Seems smooth enough for this point.

File: manuscripts/method/03.results.md [hunk 6]
```diff
@@ -9,76 +9,110 @@ MS-based global phosphoproteomic data provides unparalleled coverage when interr
...
 
-![**Validation of upstream kinase predictions.** (A-B) PCA analysis of the phosphorylation levels of MCF7 cells exposed to a drug screening. C) Upstream kinase prediction of EBDT's AKT1 and CPTAC's AKT1 cluster. D) Heatmap showing the effect of inhibitors to the phosphorylation signal of cluster 1. E) PhosphoSitePlus-reported AKT1 substrates per cluster (F) Resulting PSSM generated using ERK2 substrates reported by Carlson et al [@PMID:PMC3779841]. (G) Upstream kinase predictions of CPTAC clusters 7, 9, 13, and 21 in addition to the ERK2 motif shown in (F). H) Upstream kinase predcitions of the same PSSMs after randomly shuffling the motif positions.](figureM3.svg "Figure 3"){#fig:TvsNAT width="100%"}
+<!-- Notes figureM3:
+1. Statistical test in C?
```
Comment (aarmey) [inline manuscripts/method/03.results.md:51]: Don't bother unless reviewers ask.

File: manuscripts/method/03.results.md [hunk 7]
```diff
@@ -9,76 +9,110 @@ MS-based global phosphoproteomic data provides unparalleled coverage when interr
...
+2. G does not really resemble TBC1D5's peptide sequence... Maybe it'd be worth finding a more obvious example. Or even remove D-H? -->
```
Comment (aarmey) [inline manuscripts/method/03.results.md:52]: What's special about this particular peptide? I like everything else but this in particular seems weird about (D).

File: manuscripts/method/03.results.md [hunk 8]
```diff
@@ -9,76 +9,110 @@ MS-based global phosphoproteomic data provides unparalleled coverage when interr
...
 
-<!-- TODO: THE FACT THAT CLUSTERS 11 AND 23 ARE UPREGULATED IN NATs WOULD SUGGEST THAT SLK, PAK2, AND ERK2 ACT AS TUMOR SUPPRESSORS... Add to heatmap of upstream kinases all clusters with marked differential tumor vs NAT average abundance, not only 11, 12, and 23? General view may help explain what's going on -->
+<!-- Figure M4 notes:
+1. Mention a few peptides involved in the described biological processes
+2. I checked and cluster 11 doesn't seem to have an enrichment of fibroblast-specific genes  -->
```
Comment (aarmey) [inline manuscripts/method/03.results.md:70]: ✅

File: manuscripts/method/03.results.md [hunk 9]
```diff
@@ -9,76 +9,110 @@ MS-based global phosphoproteomic data provides unparalleled coverage when interr
...
+<!-- Notes figureM5:
+1. Show individual average abundance of peptides related to the enriched GO process in each group (WT vs Mut) instead of GO. Only keep GO of cluster 8.   
+2. I'm not sure if it's worth including the EGFRm/ALKf analysis (text below) if the figure is going to be helpful. I think it would be a bit redundant? Maybe we can show preliminary results in the supplments showing that the model is predictive but without spending as much time in interpretation as with tumor vs NAT, STK11, and infiltration?
```
Comment (aarmey) [inline manuscripts/method/03.results.md:90]: Agreed.

**Review by aarmey — APPROVED** [review]

**Comment by mcreixell** [comment]
I'll address these comments on a separate branch since it will be easier to make changes while viewing these updates in the manuscript build.

File: manuscripts/method/03.results.md (hunk 7 above)
Comment (mcreixell) [reply manuscripts/method/03.results.md:52]: It's just an example of a peptide that shows the general trends described in the first three plots. What do you find weird about (D)? @aarmey

Comment (aarmey) [reply manuscripts/method/03.results.md:52]: I think singling out any single peptide will beg the question why you picked that one.

Comment (mcreixell) [reply manuscripts/method/03.results.md:52]: The rationale for singling out a peptide is to help conceptualize the point of A-C. It shows that it follows the general trends and then plotting the motifs for each model illustrates that increasing the weight makes the cluster motif be more representative of that peptide sequence.

**Pull request merged**